motionplan/collision.go

```diff
@@ -103,15 +103,22 @@ func (oce *ObjectCollisionEntities) indexFromName(name string) (int, error) {
 	return -1, fmt.Errorf("collision entity %q not found", name)
 }
 
-func (oce *ObjectCollisionEntities) checkCollision(key, test *collisionEntity) (float64, error) {
-	distance, err := key.geometry.DistanceFrom(test.geometry)
-	return -distance, err // multiply distance by -1 so that weights of edges are positive
+func (oce *ObjectCollisionEntities) checkCollision(key, test *collisionEntity, depth bool) (float64, error) {
+	if depth {
+		distance, err := key.geometry.DistanceFrom(test.geometry)
+		return -distance, err // multiply distance by -1 so that weights of edges are positive
+	}
+	col, err := key.geometry.CollidesWith(test.geometry)
+	if col {
+		return 1, err
```

Comment:
[minor] is this being tested anywhere?  I think this might be backwards.  PS I'm sorry for making the signs complicated in this function here

Reply:
Yes; if we flip these signs then tons of our unit tests fail